src/c-writer.cc

```diff
@@ -215,7 +224,9 @@ class CWriter {
                         std::string_view module_name,
                         std::string_view field_name);
   std::string DefineGlobalScopeName(ModuleFieldType, std::string_view);
-  std::string DefineLocalScopeName(std::string_view);
+  std::string DefineLocalScopeName(std::string_view, bool);
```

Comment:
Can we give the new argument here useful name?

Reply:
Sure, it's `is_label` in the function definition. I can add a name to the declaration too if you want, although the other declarations don't seem to have them...

Reply:
I don't know why that is, we would probably add them everywhere.. but this one in particular is especially non-intuitive without a name.

Reply:
Done.